src/lua/LuaMusic.cpp

```diff
@@ -79,7 +79,7 @@ static int l_music_get_song(lua_State *l)
 static int l_music_play(lua_State *l)
 {
 	const std::string song(luaL_checkstring(l, 1));
-	bool repeat = true;
+	bool repeat = LuaPull<bool>(l, 2, false);
 	if (lua_isboolean(l, 2))
 		repeat = lua_toboolean(l, 2) != 0;
```

Comment:
This code is now redundant and can be removed... :smile: LuaPull handles pulling the boolean value from Lua.